tests.js

```diff
@@ -745,14 +1672,6 @@ window.Specs = {
 				},
 				"tests": "hsl(0,0%,0%,.5)"
 			},
-			"transparent": {
-				"links": {
-					"tr": "#transparent",
-					"dev": "#transparent",
-					"mdn": "color_value"
-				},
-				"tests": "transparent"
-			},
```

Comment:
This should be restored, since it was actually a new value for most color properties

Reply:
You're right. I've added it back.